### ISSUE TITLE
custom breakdowns feature request

### ISSUE THREAD

**Issue by gallatas** [F3tzen/interactive-sd-triangle#1]
Hello my friend, congratulations on the interactive triangle, it was really good! One question, do you intend to launch a version where the user can modify the agent numbers on their own?

**Comment by F3tzen** [comment]
yeah, I'll probably add some way to get custom breakdowns in the future, but I don't know when I'll get to implement it
